packages/@webex/plugin-cc/src/services/task/types.ts

```
@@ -0,0 +1,422 @@
+    trackingId: string;
+    mediaType: string;
+    destination?: string;
+    broadcast: boolean;
+    type: string;
+  };
+};
+
+/**
+ * Response type for the task public methods
+ */
+export type TaskResponse = AgentContact | Error | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)